### PR TITLE
Fix/unlock self-built APKs using android_dev docker

### PR DESCRIPTION
**Pull request by nirvn**
This PR fixes self-built vcpkg-based APKs using the android_dev docker's ./scripts/build.sh method.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -17,13 +17,6 @@ repos:
     - id: check-yaml
     - id: check-added-large-files
 
-# Shellcheck
-- repo: https://github.com/jumanjihouse/pre-commit-hooks
-  rev: 2.1.5
```
Comment (m-kuhn) [inline .pre-commit-config.yaml:22]: Something wrong with this?

File: scripts/build-vcpkg.sh [hunk 2]
```diff
@@ -7,6 +7,7 @@ cd /usr/src/qfield || exit
 CMAKE_BUILD_DIR=/usr/src/qfield/build-${triplet}
 
 export ANDROID_NDK_HOME=${ANDROID_NDK_ROOT}
+export Qt5_Dir=/home/devel/5.14.2/android/
 export Qt5_DIR=/home/devel/5.14.2/android/
```
Comment (m-kuhn) [inline scripts/build-vcpkg.sh:11]: Can we drop this line instead?

File: .pre-commit-config.yaml (hunk 1 above)
Comment (nirvn) [reply .pre-commit-config.yaml:22]: @m-kuhn , I couldn't get it to commit on my machine, I'll revert later. I kept getting this:

```
Test shell scripts with shellcheck.......................................Failed
- hook id: shellcheck
- exit code: 1

This check needs shellcheck from https://github.com/koalaman/shellcheck

```

File: scripts/build-vcpkg.sh (hunk 2 above)
Comment (nirvn) [reply scripts/build-vcpkg.sh:11]: @m-kuhn , sure, if only one is needed :)

File: scripts/build-vcpkg.sh [hunk 3]
```diff
@@ -24,6 +25,9 @@ cmake -S "${SOURCE_DIR}" \
       -D CMAKE_TOOLCHAIN_FILE=/usr/src/qfield/vcpkg/base/scripts/buildsystems/vcpkg.cmake \
       -D VCPKG_OVERLAY_PORTS=/usr/src/qfield/vcpkg/overlay_system_qt\;/usr/src/qfield/vcpkg/overlay \
       -D VCPKG_TARGET_TRIPLET="${triplet}" \
+      -D SYSTEM_QT=ON \
+      -D ANDROID_SDK="/home/devel/android/" \
+      -D QT_DIR="/home/devel/5.14.2/android/" \
```
Comment (m-kuhn) [inline scripts/build-vcpkg.sh:30]:
```suggestion
      -D QT_DIR="${Qt5_Dir}" \
```

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for ceb76a3f0cf98f058bd11af369bb03dedd6444dd:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-d1c7a2b33c078f7186f94ddf07fb1a633bb03118-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-d1c7a2b33c078f7186f94ddf07fb1a633bb03118-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-d1c7a2b33c078f7186f94ddf07fb1a633bb03118-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-d1c7a2b33c078f7186f94ddf07fb1a633bb03118-x86-android.apk)

File: scripts/build-vcpkg.sh (hunk 2 above)
Comment (nirvn) [reply scripts/build-vcpkg.sh:11]: We can't, just tested.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 66e1fa58eeb3a537df0cc7270505eb56d9c24051:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-34fe18811acca55b32cd1d948fb6c88e0f762db4-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-34fe18811acca55b32cd1d948fb6c88e0f762db4-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-34fe18811acca55b32cd1d948fb6c88e0f762db4-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-34fe18811acca55b32cd1d948fb6c88e0f762db4-x86-android.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 952151e9238e107bec6680e44fe6ef4d67edea11:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-cf5df67a3243ca01f14d44207ac1b55f429be42f-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-cf5df67a3243ca01f14d44207ac1b55f429be42f-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-cf5df67a3243ca01f14d44207ac1b55f429be42f-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-cf5df67a3243ca01f14d44207ac1b55f429be42f-x86-android.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 0c7302f4b1e9464596aab5d2de72a582cc2d3329:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-012e07d525b320a7b8de5a5d99282972bf9aaf53-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-012e07d525b320a7b8de5a5d99282972bf9aaf53-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-012e07d525b320a7b8de5a5d99282972bf9aaf53-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-012e07d525b320a7b8de5a5d99282972bf9aaf53-x86-android.apk)

**Review by m-kuhn — APPROVED** [review]

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for df390c1d52d3f29f404c637f2255827600217531:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-8f82171f293517e3abbc1eff8a9acf68d639f63c-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-8f82171f293517e3abbc1eff8a9acf68d639f63c-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-8f82171f293517e3abbc1eff8a9acf68d639f63c-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-8f82171f293517e3abbc1eff8a9acf68d639f63c-x86-android.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 5bf70d458e1d7d28eeaf8e2ecca5bd393a3136a5:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-d3f04029132918a7a413b79e3ab72991a2117710-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-d3f04029132918a7a413b79e3ab72991a2117710-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-d3f04029132918a7a413b79e3ab72991a2117710-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-d3f04029132918a7a413b79e3ab72991a2117710-x86-android.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 47a9a1373d825106b599029eff67e1143a6fedf9:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-c32051db281091b1bb1fe381ef1a71342860600a-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-c32051db281091b1bb1fe381ef1a71342860600a-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-c32051db281091b1bb1fe381ef1a71342860600a-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2627-c32051db281091b1bb1fe381ef1a71342860600a-x86-android.apk)

**Pull request merged**